projects/storefrontlib/cms-components/misc/message/message.component.ts

```diff
@@ -67,4 +80,15 @@ export class MessageComponent {
         return ICON_TYPE.SUCCESS;
     }
   }
+
+  ngAfterViewInit(): void {
+    // TODO: (CXSPA-7725) - Remove feature flag next major release.
+    if (
+      (this.featureConfigService.isEnabled('a11yCxMessageFocus') &&
```

Comment:
It looks like the brackets might be in the wrong place here. I think this would fire regardless of the feature flag setting.

Reply:
On second look, `#messageContainer` will only exist when the feature flag is active anyway. Can we just use this instead?

```
if(this.accordianText || this.actionButtonText) {
  this.messageContainer?.nativeElement?.focus();
}
```

Note: I'm not sure if the `?.` is needed in both places here.

I think we could also remove `private featureConfigService = inject(FeatureConfigService);` above if this is the case.

Reply:
You are right, the template feature flag is enough.